docs/conf/portland/2019/hike.rst

```diff
@@ -12,9 +12,8 @@ It's rained on us the past, but we have faith it will be beautiful this year! We
 Logistics
 ---------
 
-.. TODO
 
-.. You can get your hike ticket here: https://ti.to/writethedocs/write-the-docs-portland-2018/with/awor0lxyamc
+**You can get your hike ticket here**: https://ti.to/writethedocs/write-the-docs-portland-2019/with/dn2-wy5dqta
```

Comment:
Maybe it's just me, but I missed this detail the first time. 

here's one attempt to make the ticket link more visible:

`Get Your Hike Ticket at This Link <https://ti.to/writethedocs/write-the-docs-portland-2019/with/dn2-wy5dqta>`__
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~